### ISSUE TITLE
A11y_VisualStudioCodeClient_Editor_StatusMessages: Screen reader does not announce confirmation of the folding and unfolding action of code.

### ISSUE THREAD

**Issue by kupatkar99** [microsoft/vscode#141715]
### GitHubTags:
#A11yMAS; #A11yTCS; #A11ySev2; #DesktopApp; #Visual Studio Code Client; #BM-VisualStudioCode-Win32-Nov2021; #Win32; #Narrator;#NVDA; #ScreenReader; #StatusMessages;#WCAG4.1.3; #AINotInScope; #FTP; #STP;

### Environment Details:
Application Name: Visual Studio Code Client
Windows Version: Windows 11

### Repro Steps:
1)Open Visual studio code editor    
2)Create an "index.html" file and type "!" + TAB key to create a code snippet with Emmet Syntax.
3)Create a CSS file with selectors/elements and use keyboard navigation to see how the hierarchy of rules are applied (can be done 
  by hovering over selectors with a mouse)
4)With keyboard bindings and other ATs in scope, use "multi-cursor editing", "code folding", "bracket matching" to style code.
5)Verify that screen reader is announcing the confirmation of the folding and unfolding action of code or not.

### Actual Result:
Screen reader does not announce confirmation of the folding and unfolding action of code.

### Expected Result:
Screen reader should announce the confirmation of the folding and unfolding action of code.

### User Impact:
Screen Reader users will not understand the status of the action whether the code is folded or not.

**Comment by isidorn** [comment]
Fixed in latest insiders. We introduced sound to announce if a region is folded
